stdlib/3.4/asyncio/locks.pyi

```diff
@@ -49,11 +51,11 @@ class Condition(_ContextManagerMixin):
     def notify_all(self) -> None: ...
 
 class Semaphore(_ContextManagerMixin):
-    def __init__(self, value: int = ..., *, loop: Optional[AbstractEventLoop] = ...) -> None: ...
+    def __init__(self, value: int = 1, *, loop: AbstractEventLoop = None) -> None: ...
```

Comment:
This looks odd. Why did you change `...` to "1" and "None"? We generally prefer the former, in typeshed.